packages/gatsby-cli/src/reporter/redux/action-types.ts

```diff
@@ -0,0 +1,138 @@
+import { Actions, ActivityStatuses } from "../constants"
+import { IActivity } from "./reducer"
+import { Dispatch, AnyAction } from "redux"
+
+type dispatchAction = (dispatch: Dispatch<AnyAction>) => void
+
+export type actionsToEmitType = Array<
+  dispatchAction | { type: Actions; payload: Partial<IActivity> }
+>
+
+export interface IActionTypes {
```

Comment:
I don't see the need for this to be reusable (in the sense of being an export in a common file). Could you just transfer these types to exist on the functions themselves?

Reply:
Do you mean to export them types manually instead of an interface? I put them in a different file cause it grew quite a lot.